### PR TITLE
[DO NOT MERGE] CircleCI to release iOS framework when merging into master

### PR DESCRIPTION
**Pull request by ferrannp**

### CODE REVIEW

File: scripts/releaseFramework.js [hunk 1]
```diff
@@ -10,34 +12,39 @@ const child_process = require('child_process');
...
-          name: version,
-          draft: false,
-          prerelease: false,
-        }),
-      }
-    )
-  ).json();
+  const release = await (await fetch(
+    `https://api.github.com/repos/kiwicom/mobile/releases?access_token=${
+      // $FlowFixMe env.GITHUB_TOKEN will be a string
```
Comment (mrtnzlml) [inline scripts/releaseFramework.js]: Fix it directly please - there is a reason for this Flow error. See for example: https://github.com/facebook/flow/issues/1192

Comment (ferrannp) [reply scripts/releaseFramework.js]: Ok once I have the PR actually working (WIP) I'll do that.

**Comment by mrtnzlml** [comment]
@ferrannp I noticed the build (release job) is green even though it's failing. It would be great to make it red so we know something is wrong... 🙂

**Comment by ferrannp** [comment]
As we already have the script in `master`, let's do this in separated PR.

**Pull request closed without merging**